src/charactersheet/services/common/rng_service.js

```diff
@@ -0,0 +1,109 @@
+import {SharedServiceManager} from './shared_service_manager';
+var Random = require('random-js');
```

Comment:
Could you change this to the es6 style import statements? 

`import Random from 'random-js';`